.github/workflows/docs-ci.yml

```diff
@@ -0,0 +1,53 @@
+name: Docs Continuous Integration
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
```suggestion
on:
  push:
    branches:
      - main
```

Reply:
if you do that you lose the nice docs checks report on your pr (which is the whole point of this CI action...) 
this is like a linting for your docs... I dont think you want to skip it...
example:  https://github.com/TBD54566975/web5-js/pull/269#issuecomment-1802976855

Reply:
Cool!